src/DocumentationBrowserViewExtension/MarkdownHandler.cs

```diff
@@ -1,26 +1,23 @@
 ﻿using System;
 using System.IO;
-using System.Linq;
-using Dynamo.PackageManager;
-using Markdig;
-using Markdig.Parsers;
-using Markdig.Renderers;
-using Markdig.Syntax;
-using Markdig.Syntax.Inlines;
+using Dynamo.Utilities;
 
 namespace Dynamo.DocumentationBrowser
 {
     /// <summary>
     /// Handles markdown files by converting them to Html, so they can display in the doc browser.
     /// </summary>
-    internal class MarkdownHandler
+    internal class MarkdownHandler : IDisposable
```

Comment:
Might be worthwhile to mention that this is supposed to be a Singleton

Reply:
Done